src/components/add-mfa-method/tests/add-mfa-methods-controller.test.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from "chai";
+import { describe, it } from "mocha";
+import { Request, Response } from "express";
+import sinon, { SinonSandbox } from "sinon";
+import { addMfaMethodPost } from "../add-mfa-methods-controller";
+import { PATH_DATA } from "../../../app.constants";
+
+describe("addMfaMethodPost", () => {
+  let sandbox: SinonSandbox;
+  let req: Partial<Request>;
+  let res: Partial<Response>;
+  let next: sinon.SinonSpy;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+    req = { body: {}, log: { error: sandbox.fake() } as any };
+    res = {
+      status: sandbox.fake(),
+      end: sandbox.fake(),
+      redirect: sandbox.fake(),
+    };
+    next = sinon.spy();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it("should return 400 status code when addMfaMethod is 'sms'", () => {
```

Comment:
Could the name of the test describe what happens from the users perspective?  e.g. should not allow user to add a backup MFA method of APP when the primary method is SMS.

Reply:
agreed and changed